packages/SidePanel/src/components/Header/Header.js

```diff
@@ -31,24 +33,27 @@ function darkBackgroundProps(kind) {
 const Header = React.forwardRef((props, ref) => {
   const {
     hasCloseButton,
-    kind,
     isCompact,
+    kind,
+    level,
     // injected by Dialog.js
     onClose,
     ...moreProps
   } = props;
 
   return (
     <div data-pka-anchor="sidepanel.header" ref={ref} css={headerCSS} kind={kind} isCompact={isCompact} {...moreProps}>
-      <div tabIndex="-1">{props.children}</div>
+      <Heading tabIndex="-1" level={level || 2}>
```

Comment:
Why `level || 2` if `level` is required?

Reply:
True, i should not make it required above then, and just make it defaulted to `2`